packages/edit-site/src/components/canvas-loader/style.scss

```diff
@@ -16,6 +16,10 @@
 	& > div {
 		width: 160px;
 	}
+
+	& > .edit-site-canvas-loader__progressbar {
+		width: 160px;
```

Comment:
Is there a good reason to prefer `width` to `max-width`?

Reply:
Actually no, we should indeed probably use `max-width` so that it adapts to smaller viewports. Thanks for catching that 👍🏻